src/theme/io-sanita/_main.scss

```diff
@@ -279,3 +279,27 @@ picture.volto-image.responsive img.full-width,
   margin: 0;
   box-shadow: none;
 }
+
+//Fix for photo gallery alignament
+#main-content-section {
+  .slider-container {
+    .slick-track {
+      @media (min-width: #{map-get($grid-breakpoints, sm)}) {
+        //if the gallery have more than 1 item, this rule will be applied to set alignment
+        .slick-slide:not(:only-child) {
+          figure {
+            picture,
+            a {
+              height: 180px;
+              img {
+                width: 100%;
+                height: 100%;
+                object-fit: cover;
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
queste regole spostale in un file dedicato gallery.scss che metti sotto al path io-sanita-theme/src/components/View/commons/Gallery

Reply:
fatto